한단비/movie/movie-react/src/components/Banner.jsx

```diff
@@ -1,9 +1,90 @@
-import React from 'react'
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  width: 100%;
+  height: calc(100vh - 64px);
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+`;
+const BannerContainer = styled.div`
+  width: 100%;
+  height: 40vh;
+  background-color: black;
+  display: flex;
+  justify-content: center;
+`;
+
+const BannerText = styled.h1`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: 25px;
+  font-weight: bold;
+  text-align: center;
+`;
+
+const SearchContainer = styled.div`
+  width: 100%;
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #1F2141;
+`;
+
+const SearchTitle = styled.h1`
+  width: 40%;
+  text-align: center;
+  font-size: 30px;
+  color: white;
+`
+
+const InputContainer = styled.div`
+  width: 50%;
+  height: 10vh;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+`;
+
+const SearchInput = styled.input`
+  width: 30%;
+  height: 25px;
+  border-radius: 25px;
+  border: none;
+  padding: 10px;
+`;
+
+const SearchIcon = styled.button`
+  width: 24px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #FEC623;
+  border: none;
+  border-radius: 30px;
+`
 
 export default function Banner() {
   return (
-    <div className='banner-container'>
-        <div className='banner-text'>환영합니다</div>
-    </div>
-  )
+    <Container>
+      <BannerContainer>
+        <BannerText>환영합니다</BannerText>
+      </BannerContainer>
+      <SearchContainer>
+        <SearchTitle>📽 Find your movies!</SearchTitle>
+        <InputContainer>
+          <SearchInput></SearchInput>
```

Comment:
진짜 사소한건데 태그 안에 아무것도 없으면 이왕이면 셀프클로징태그를 사용하는게 국룰아닌 국룰인것으로 알아요ㅎㅎㅎ (닫는 태그가 없는 태그 제외)
```suggestion
          <SearchInput />
```

Reply:
감사합니당 !!